data/styles/clear/include/Icons.mapcss

```diff
@@ -1606,6 +1606,12 @@ area|z17[amenity=dentist]
 node|z18-[amenity=dentist],
 area|z18-[amenity=dentist]
 {icon-image: dentist-m.svg;font-size: 11;text-optional: true;}
+node|z17[healthcare=physiotherapist],
+area|z17[healthcare=physiotherapist]
+{icon-image: physiotherapist-m.svg;}
+node|z18-[healthcare=physiotherapist],
+area|z18-[healthcare=physiotherapist]
+{icon-image: physiotherapist-m.svg;font-size: 11;text-optional: true;}
```

Comment:
Add healthcare=physiotherapist to other places in this file, exactly like healthcare=laboratory.
It should also have {text: name} style.

Reply:
I added the entries in `data/styles/clear/include/Icons.mapcss` and regenerated the styles.
However on my locally built map, I still do not see anything.

<details>
<summary>What I did</summary>

1. `tools/unix/generate_symbols.sh`
2. `tools/unix/generate_drules.sh`
3. `python -m maps_generator --countries="Germany_Berlin" --skip="Coastline"`
4. Copied the result to the phone
</details>